src/components/TeamSection/TeamInformation.ts

```diff
@@ -67,62 +69,79 @@ executiveTeam = [
   },
   {
     id: 4,
+    name: "Noel Thomas",
+    affiliation: "Co-VP Events",
```

Comment:
Is this correct @Rajpreetgill or are we going with Events Commissioner per the constitution?

Reply:
This is correct, I'll update the constitution to reflect this change but Co-VP Events is the correct title. thanks!